spec/models/user_spec.rb

```diff
@@ -33,7 +33,8 @@
       user_sp = create(:service_provider, user: user)
       team_sp = create(:service_provider, team: team)
       create(:service_provider)
-      expect(user.scoped_service_providers).to eq([user_sp, team_sp])
+      sorted_sps = [user_sp, team_sp].sort_by { |x| x.friendly_name.downcase }
+      expect(user.scoped_service_providers).to eq(sorted_sps)
```

Comment:
We can use RSpec's `match_array` to simplify this (it does order-independent comparison)

```suggestion
      expect(user.scoped_service_providers).to match_array([user_sp, team_sp])
```

some examples: https://relishapp.com/rspec/rspec-expectations/docs/built-in-matchers

Reply:
That's what I had initially, but Andrew caught that `scoped_service_providers` is actually sorted by `friendly_name`, so I changed the test to match that behavior

Reply:
Oh ok! sounds good